library/alloc/src/collections/btree/node/tests.rs

```diff
@@ -23,3 +23,17 @@ fn test_splitpoint() {
         assert!(left_len + right_len == CAPACITY);
     }
 }
+
+#[test]
+fn test_sizes() {
+    #[cfg(all(any(target_os = "windows", target_os = "unix"), target_pointer_width = "32"))]
```

Comment:
We should be able to get away with just `#[cfg(target_arch = "x86_64)]` and a single assert, I think -- that's what we do in rustc and it works well.

Reply:
Yeah, that seems wiser. I actually wandered what the effect is of the two `u16` fields in `LeafNode` that are combined - if it helps to give them each 32 bits on 64 bit platforms. But testing that doesn't have to remain in there.